Makefile

```diff
@@ -18,6 +18,22 @@ generate-accounts:
 
 run:
 	# Run containers
+	docker run -d --name mail_hostoverride \
```

Comment:
Could you use an existing container like `mail_smtponly`?
It avoids to launch to many of them consuming CPU and memory.
Otherwise it's ok!
Thanks.